modules/cluster-and-ha-management/pages/region-aware.adoc

```diff
@@ -59,7 +59,25 @@ By enabling region awareness, the service deployments would be structured as fol
 |  GPE_5  | us-east-3:[m10], us-east-1:[m7]
 |===
 
-In this setup, a failure in any region would retain at least one replica of GPE for each partition and the majority count of ZK/ETCD replicas.
+In this setup, a failure in any region/zone would retain at least one replica of GPE for each partition and the majority count of ZK/ETCD replicas.
+
+== Compare to CRR
+
+Region-Awareness **is not**  xref:crr-index.adoc[Cross-Region Replication].
+
+Both Region-Aware and CRR can increase your system availability, but they are designed for different purpose.
+
+.Differences between Region-Aware and CRR
+|===
+|             | **Region-Aware**     |   **CRR**
+|Cluster Setup| 1 read-write   cluster  | 1 read-write cluster, other read-only cluster
+|Minimum nodes|  3                   | 2
+|Fail over    | Auto recover         | Manual switch
+|===
+
+Overall, if you are looking for zero downtime region failure toleration, use region-awareness. If you are looking for data residency compliance or Blue/green deployment, use CRR.
```

Comment:
https://github.com/tigergraph/server-docs/pull/452#discussion_r1717438286 as stated here, we need to mention region-aware deployment can have high latencies.